fields/types/gcsavatar/GcsAvatarType.js

```diff
@@ -0,0 +1,378 @@
+/*!
+ * Module dependencies.
+ */
+var fs = require('fs');
+var gcsHelper = require('../../../lib/gcsHelper');
+var keystone = require('../../../');
+var moment = require('moment');
+var super_ = require('../Type');
+var util = require('util');
+var utils = require('keystone-utils');
+
+// lodash
+var forEach = require('lodash/forEach');
+var indexOf = require('lodash/indexOf');
+var set = require('lodash/set');
+
+const _ = {
+  forEach,
+  indexOf,
+  set,
+};
+
+/**
+ *  FieldType Constructor
+ * @extends Field
+ * @api public
+ */
+
+function gcsavatar(list, path, options) {
+  this._underscoreMethods = ['format', 'uploadFile'];
+  this._fixedSize = 'full';
+
+  options.nofilter = true;
+
+  gcsavatar.super_.call(this, list, path, options);
+
+  // validate gcs config (has to happen after super_.call)
+  if (!keystone.get('gcs config')) {
+    throw new Error('Invalid Configuration\n\n' + 'gcsavatar fields (' + list.key + '.' + path + ') require the "gcs config" option to be set.\n\n' + 'See http://keystonejs.com/docs/configuration/#services-gcsavatar for more information.\n');
+  }
```

Comment:
Do we update the doc as well or I believe the anchor is invalid `http://keystonejs.com/docs/configuration/#services-gcsavatar`

Reply:
Current type usage is inconsistent with the services in the original doc.
I'm afraid it might take lots of effort to update the doc to meet current status in this project.

I prefer to remove the doc link as it is an invalid one.

Reply:
I'm second to remove the doc.